cachito/workers/config.py

```diff
@@ -253,6 +256,27 @@ def validate_celery_config(conf, **kwargs):
             )
 
 
+def populate_ssh_known_hosts(conf, **kwargs):
+    """
+    Populate ~/.ssh/known_hosts based on some predefined list of hosts.
+
+    The list of hosts can be provided using `cachito_git_ssh_known_hosts` configuration option.
```

Comment:
Does this really protect against a MitM attack? How often does this get populated? On every deployment?

It feels like it would be "safer" to cache off the known_hosts _now_ and then just populate that. That way, we can ensure that it isn't changing without our knowledge.

Reply:
> to cache off the known_hosts now and then just populate that

In the case, we can just mount the the ssh config and corresponding known_hosts file during the service configuration and pass them through `GIT_SSH_COMMAND` env variable (or `core.sshCommand` git option). So that patch is not needed at all.

WDYT, @brunoapimentel , @arewm ?

Reply:
Yes, that works for me. It will also (presumably) allow any deployments to customize the known ssh hosts if there are also any non-public systems that they want to allow access to.

Reply:
Makes sense to me. I think the known_hosts file can be added to the cachito-playbooks and mounted during the deployment, as you mentioned.